packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -170,6 +170,19 @@ export const diffusers = (model: ModelData): string[] => {
 	}
 };
 
+export const dynamicrafter = (): string[] => [
+	`# Git clone https://github.com/Doubiiu/DynamiCrafter.git,
+# Download pretrained models via Hugging Face, 
+# and put the model.ckpt with the required resolution in 
+# checkpoints/dynamicrafter_[1024|512|256]_v1/model.ckpt
+
+# Run on a single GPU:
+# Select the model based on required resolutions: i.e., 1024|512|320:
+sh scripts/run.sh 1024
+# Run on multiple GPUs for parallel inference:
+sh scripts/run_mp.sh 1024`,
+];
+
```

Comment:
Same comment as in https://github.com/huggingface/huggingface.js/pull/805#discussion_r1680623707. Better to either focus on download count and remove the code snippet or dynamically render the snippet depending on the model id + use `huggingface-cli download` (instead of the generic _Download pretrained models via Hugging Face, and put the model.ckpt with the required resolution in checkpoints/dynamicrafter_[1024|512|256]_v1/model.ckpt_)

Reply:
Ok, removed the snippet